conventions/build.gradle.kts

```diff
@@ -11,8 +11,6 @@ spotless {
     target("src/**/*.java")
   }
   kotlinGradle {
-    // not sure why it's not using the indent settings from .editorconfig
-    ktlint().editorConfigOverride(mapOf("indent_size" to "2", "continuation_indent_size" to "2", "disabled_rules" to "no-wildcard-imports"))
```

Comment:
So `ktlint()` is applied by default now? I couldn't find a reference to that in the spotless [changelog](https://github.com/diffplug/spotless/blob/main/plugin-gradle/CHANGES.md#6110---2022-09-14).

Reply:
oops! I thought I was just removing the editorConfig properties, thx for the catch